pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -231,6 +231,21 @@ class MindtPySolver(object):
                     "slack variables here are used to deal with nonconvex MINLP",
         domain=bool
     ))
+    CONFIG.declare("continuous_var_bound", ConfigValue(
+        default=1e10,
+        description="default bound added to unbounded continuous variables in nonlinear constraint if single tree is activated.",
+        domain=PositiveFloat
+    ))
+    CONFIG.declare("integer_var_bound", ConfigValue(
+        default=1e9,
+        description="default bound added to unbounded integral variables in nonlinear constraint if single tree is activated.",
+        domain=PositiveFloat
+    ))
+    CONFIG.declare("cycling_check", ConfigValue(
+        default=True,
+        description="whether check the cycling in OA algorithm.",
```

Comment:
Suggested: "Check if OA algorithm is stalled in a cycle and terminate."

I assume terminate, but this doesn't actually say.